src/main/java/com/_up/megastore/controllers/interfaces/ISizeController.java

```diff
@@ -18,4 +18,6 @@ public interface ISizeController {
     @PutMapping (value = "/{sizeId}") @ResponseStatus(HttpStatus.OK)
     SizeResponse updateSize(@PathVariable UUID sizeId, @RequestBody @Valid UpdateSizeRequest updateSizeRequest);
 
+    @PostMapping (value = "{/sizeId}/restore") @ResponseStatus(HttpStatus.OK)
```

Comment:
Esto está mal escrito. Si encerrás a la barra dentro de las llaves el endpoint queda inválido. Esto compila pero Spring no es capaz de mapear el endpoint por lo que la aplicación no corre
```suggestion
    @PostMapping (value = "/{sizeId}/restore") @ResponseStatus(HttpStatus.OK)
```